pkg/app/run.go

```diff
@@ -12,8 +12,9 @@ import (
 	"github.com/fastly/cli/pkg/commands/version"
 	"github.com/fastly/cli/pkg/config"
 	"github.com/fastly/cli/pkg/env"
-	"github.com/fastly/cli/pkg/errors"
+	fsterr "github.com/fastly/cli/pkg/errors"
```

Comment:
We should never have imported our own `errors` package without a custom namespace because it gets very confusing when you actually want to use the go standard library's own `errors` package.

So whenever I stumble across code that doesn't use the `fsterr` custom namespace I make sure to update it.